reactor-kafka-samples/src/main/java/reactor/kafka/samples/SampleScenarios.java

```diff
@@ -398,9 +398,9 @@ public FanOut(String bootstrapServers, String sourceTopic, String destTopic1, St
             return Flux.merge(stream1, stream2)
                        .doOnSubscribe(s -> cancelRef.set(inFlux.subscribe()))
                        .doOnCancel(() -> {
-                               cancel.accept(cancelRef);
-                               close();
-                           });
+                           cancel.accept(cancelRef);
+                           close();
```

Comment:
Not related to the indentation change, but shouldn't `close` go to `doFinally`?

Reply:
done